internal/uvm/types.go

```diff
@@ -82,6 +82,8 @@ type UtilityVM struct {
 	scsiLocations       [4][64]*SCSIMount // Hyper-V supports 4 controllers, 64 slots per controller. Limited to 1 controller for now though.
 	scsiControllerCount uint32            // Number of SCSI controllers in the utility VM
 
+	vpciIDs map[string]*VPCIDevice // map of device instance id to vpci device
```

Comment:
I think `vpciDevices` is a better name for this, as it's mostly about tracking the devices, they just happen to be indexed by ID.

Reply:
Done.